roles/openshift_azure/defaults/main.yml

```diff
@@ -0,0 +1,76 @@
+---
+openshift_azure_clusterid: "{{ openshift_clusterid | default('default') }}"
+
+openshift_azure_image_build_base: "{{ not (openshift_node_bootstrap | default(False)) }}"
+openshift_azure_base_image: "{{ openshift_azure_clusterid }}_base-{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}"
+openshift_azure_image: "{{ openshift_azure_clusterid }}-{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}"
+openshift_azure_image_tags: {}
+
+
+openshift_azure_create_resource_group: True
+openshift_azure_resource_group_name: "{{ openshift_azure_clusterid }}"
+
+openshift_azure_resource_location: eastus
+
+openshift_azure_storage_account_name: "{{ openshift_azure_clusterid }}"
+openshift_azure_storage_account_container: images
+openshift_azure_storage_account_type: Standard_LRS
+openshift_azure_subnet_prefix: 172.31.48.0/20
+openshift_azure_virtual_network_address_prefixes:
```

Comment:
what are these specified for?

Reply:
These are just default variables so one can have the shortest path to running the code without having to specify variables.

Reply:
For default provisioning.  Makes life easier.